src/gui/about_imp.h

```diff
@@ -36,6 +36,11 @@
 #include "base/utils/misc.h"
 #include "base/unicodestrings.h"
 
+#define SIXTYFOUR ""
```

Comment:
Mark 32 bit explicitly, ugly but reduce ambiguous, @evsh are you going to commit it?
```c++
QLatin1String(VERSION) + (sizeof(void*) == 8 ? QLatin1String(" (64 bit)") : QLatin1String(" (32 bit)")
```